-1

Tested at: cfa0ba11a54302a957cb42ff7e55bcb2e74e6d9b

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15205/14072/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/IRPCDigitizer.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCandIRPCDigiProducer.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCSimModelTiming.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCSim.cc 
In file included from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/MCTruth/plugins/SealModule.cc:3:0:
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/poison/SimMuon/MCTruth/src/CSCTruthTest.h:1:2: error: #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
 #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
  ^
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCDigitizer.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCSimParam.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-07-14-1100/src/SimMuon/RPCDigitizer/src/RPCSimSetUp.cc 
</pre>

--- a/SimMuon/MCTruth/interface/CSCTruthTest.h
+++ b/SimMuon/MCTruth/interface/CSCTruthTest.h
@@ -1,9 +1,12 @@
-// user include files
+#ifndef MCTruth_CSCTruthTest_h
+#define MCTruth_CSCTruthTest_h
+
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/stream/EDAnalyzer.h"
 #include "SimMuon/MCTruth/interface/MuonTruth.h"
 
 class CSCTruthTest : public edm::stream::EDAnalyzer<> {
+
 public:
   explicit CSCTruthTest(const edm::ParameterSet&);
   virtual ~CSCTruthTest();
@@ -15,3 +18,4 @@ private:
 
 };
 
+#endif

--- a/SimMuon/MCTruth/plugins/SealModule.cc
+++ b/SimMuon/MCTruth/plugins/SealModule.cc
@@ -1,6 +1,6 @@
 #include "SimMuon/MCTruth/plugins/MuonAssociatorEDProducer.h"
 #include "SimMuon/MCTruth/plugins/MuonTrackProducer.h"
-#include "SimMuon/MCTruth/src/CSCTruthTest.h"
+#include "SimMuon/MCTruth/interface/CSCTruthTest.h"
 #include "FWCore/PluginManager/interface/ModuleDef.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
 

--- a/SimMuon/MCTruth/src/CSCTruthTest.cc
+++ b/SimMuon/MCTruth/src/CSCTruthTest.cc
@@ -1,5 +1,4 @@
 #include "SimMuon/MCTruth/interface/CSCTruthTest.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "DataFormats/CSCRecHit/interface/CSCRecHit2DCollection.h"

